### ISSUE TITLE
Barrels and ropes.

### ISSUE THREAD

**Issue by cotin18** [NextdoorPsycho/FulBlkPublicRequests#3]
https://io.conquestreforged.com/idlist/ and search barrel. I wan ALL those blocks in that list.

Vertical and horizontal ropes. similar to redstone wire blocks. (like in conquest reforged)

**Comment by NextdoorPsycho** [comment]
![image](https://user-images.githubusercontent.com/29320316/42468051-8ea3177c-8368-11e8-8a40-2864d849ad1b.png)
and
![image](https://user-images.githubusercontent.com/29320316/42468065-9965f9d6-8368-11e8-9b63-7deec9111462.png)
Correct?

**Comment by NextdoorPsycho** [comment]
got any textures? haha

**Comment by NextdoorPsycho** [comment]
got any textures? haha

**Comment by cotin18** [comment]
That is correct :D. And no i dont have any textures... BUT i know photoshop and a little illustrator. Soo i guess i could try to help with that!
What resolution should the textures be in?

edit: merged

**Comment by cotin18** [comment]
What resolution should the textures be in?

**Comment by NextdoorPsycho** [comment]
what do you want them in? 32x32 64x64 16x16 anything works just lemme know i was using  16x16 for this
![barrel_side](https://user-images.githubusercontent.com/29320316/42468871-e9c1b184-836a-11e8-8646-6a51dc1a049d.png)

**Comment by cotin18** [comment]
Hmm well.. The resolution should be the same through out the whole texturepack. You dont want 16x16 grass blocks and 64x64 wood blocks. And for the sake of our players i wouldn't use anything higher than 32x32. What do you think?

**Comment by NextdoorPsycho** [comment]
yee so lets do 32

**Comment by cotin18** [comment]
sounds good.
